assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -113,6 +114,7 @@ function InspectorProvider({ children }) {
 
   const state = {
     state: {
+      inspectorRef,
```

Comment:
WDYT if we had an agreement to place these to the `refs: {inspector: ...}` instead of `state`? Ok to skip for now.